cas-server-webapp/src/main/webapp/WEB-INF/webflow/login/login-webflow.xml

```diff
@@ -110,7 +110,8 @@
         <transition on="FailedLoginException" to="generateLoginTicket"/>
         <transition on="AccountNotFoundException" to="generateLoginTicket"/>
         <transition on="UnauthorizedServiceForPrincipalException" to="generateLoginTicket" />
-        <transition on="UNKNOWN" to="generateLoginTicket"/>
```

Comment:
I replaced it with a catch-all transition that handles any event not explicitly defined.

This will allow anyone that wants to override which AuthenticationExceptions are handled an easier integration if they just want the custom error message and not any custom webflow states. I think leaving the catch-all is good for deployers, but I can restore the `UNKNOWN` transition if we want that explicitly defined.